tmVhdlProducer/main.py

```diff
@@ -35,6 +35,8 @@
     'ext': [kExternals],
     'zdc': [kZDCPlus, kZDCMinus],
 }
+
+DefaultModuleZdc = 0
 """Mapping constraint types to esCondition types, provided for convenience."""
```

Comment:
Take care, the doc string was attached to `ConstraintTypes` and is now assigned to `DefaultModuleZdc`!

Reply:
Moved "DefaultModuleZdc = 0" up. Done!